config/crd/bases/servicemonitor_crd.yaml

```diff
@@ -0,0 +1,1230 @@
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.80.0/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Shouldn't the enduser bring the servicemonitor CRD together with its prometheus-operator installation?

Reply:
yes I removed them, it should ideally be installed with prometheous-operator, However, in my local cluster, this was somehow giving me an error for this missing CRD so I had added it explicitly.